MIDIchlorians/MIDIchlorians/PlayBackRetriever.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  PlayBacker.swift
+//  MIDIchlorians
+//
+//  Created by Ten Zhi yang on 3/4/17.
+//  Copyright © 2017 nus.cs3217.a0118897. All rights reserved.
+//
+
+import Foundation
+
+struct PlayBackRetriever {
+    private var timeIndexArr: [(TimeInterval, (Int, IndexPath))]
+    init(timeIndexArr: [(TimeInterval, (Int, IndexPath))]) {
+        self.timeIndexArr = timeIndexArr
+    }
+
+    //returns empty arr if there's nothing left
+    //otherwise, returns one or more tuples
+    mutating func getNextPads() -> [(TimeInterval, (Int, IndexPath))] {
```

Comment:
what is this method trying to do actually, is it returning all pads that have the same time interval?